src/ast/seq_decl_plugin.cpp

```diff
@@ -1449,3 +1449,17 @@ bool seq_util::re::is_loop(expr const* n, expr*& body, expr*& lo) const {
     }
     return false;
 }
+
+/**
+   Returns true iff e is the epsilon regex.
+ */
+bool seq_util::re::is_epsilon(expr* r) const {
+    expr* r1;
```

Comment:
maybe call this s, because the type is a _sequence_ or _string_, not a regex

Reply:
you mean r1? here r is a regex

Reply:
expr* s = nullptr;
return is_to_re(r, s) && u.str.is_empty(s);

Reply:
Yes this is my mistake, r1 should be named s